app/src/main/res/layout/app_update_card.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The design for this has a corner radius for this, should we wrap this with a card view or set a corner radius background?